src/app/theme-brand.css

```
@@ -1,2 +1,2 @@
-.brand-compact{color:#07101d!important}.brand-compact .brand-mark-image{width:38px!important;height:38px!important;display:block!important;opacity:1!important;filter:none!important}.brand-compact .brand-wordmark{display:inline-block!important;color:#071a35!important;-webkit-text-fill-color:#071a35!important;background:none!important;opacity:1!important;font-family:'Manrope',system-ui,sans-serif!important;font-size:21px!important;font-weight:800!important;letter-spacing:-.045em!important;line-height:1!important;text-shadow:none!important}.brand-compact:hover .brand-wordmark{color:#173d70!important;-webkit-text-fill-color:#173d70!important}.topbar .brand-compact:focus-visible{outline:3px solid rgba(23,61,112,.18)!important;outline-offset:4px!important;border-radius:10px!important}
-:root[data-theme="dark"] .brand-compact .brand-wordmark{color:#f7faff!important;-webkit-text-fill-color:#f7faff!important}.brand-compact .brand-mark-image{border-radius:10px}
+.brand-compact{color:var(--sn-ink)!important}.brand-compact .brand-mark-image{width:38px!important;height:38px!important;display:block!important;opacity:1!important;filter:none!important}.brand-compact .brand-wordmark{display:inline-block!important;color:var(--sn-ink)!important;-webkit-text-fill-color:var(--color-text-primary)!important;background:none!important;opacity:1!important;font-family:'Manrope',system-ui,sans-serif!important;font-size:21px!important;font-weight:800!important;letter-spacing:-.045em!important;line-height:1!important;text-shadow:none!important}.brand-compact:hover .brand-wordmark{color:var(--sn-ink)!important;-webkit-text-fill-color:var(--color-text-secondary)!important}.topbar .brand-compact:focus-visible{outline:3px solid var(--sn-line)!important;outline-offset:4px!important;border-radius:10px!important}
```

Stylelint (17.14.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)